cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -1,119 +1,459 @@
+        data = json.loads(data)  
+        custom_field = json.loads(custom_field)  # Ensure custom_field is parsed as a list
+
+        salary_component = data.get("salary_component")
+
+        if len(component) > 0 and len(custom_field) > 0 and salary_component:
+            get_salary_component = frappe.get_list(
+                'Salary Component',
+                filters={
+                    "name": salary_component,
+                    "disabled": 0,
+                    "type": data.get("component_type")  # Corrected variable reference
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)